packages/contracts/src/abis/PublicLock/PublicLockV9.json

```diff
@@ -1781,8 +1805,8 @@
016020860161597f565b601f01601f19169290920160200192915050565b6000825161556181846020870161597f565b9190910192915050565b6000855161557d818460208a0161597f565b855190830190615591818360208a0161597f565b85519101906155a481836020890161597f565b84519101906155b781836020880161597f565b019695505050505050565b7f416363657373436f6e74726f6c3a206163636f756e74200000000000000000008152600083516155fa81601785016020880161597f565b7001034b99036b4b9b9b4b733903937b6329607d1b601791840191820152835161562b81602884016020880161597f565b01602801949350505050565b6001600160a01b03888116825287811660208301528616604082015260c06060820181905260009061566c90830186886154fa565b60808301949094525060a0015295945050505050565b6001600160a01b038581168252848116602083015283166040820152608060608201819052600090612f4390830184615523565b6001600160a01b0385811682528416602082015260408101839052608060608201819052600090612f4390830184615523565b6020815260006143f36020830184866154fa565b6020815260006115b86020830184615523565b6020808252600f908201526e494e56414c49445f4144445245535360881b604082015260600190565b6020808252600d908201526c12d15657d393d517d590531251609a1b604082015260600190565b6020808252601c908201527f4f4e4c595f4b45595f4d414e414745525f4f525f415050524f56454400000000604082015260600190565b6020808252818101527f4f4e4c595f4c4f434b5f4d414e414745525f4f525f42454e4546494349415259604082015260600190565b60208082526035908201527f4d6978696e526f6c65733a2063616c6c657220646f6573206e6f74206861766560408201527420746865204c6f636b4d616e6167657220726f6c6560581b606082015260800190565b6020808252600f908201526e1313d0d2d7d11154149150d0551151608a1b604082015260600190565b6020808252602b908201527f496e697469616c697a61626c653a20636f6e7472616374206973206e6f74206960408201526a6e697469616c697a696e6760a81b606082015260800190565b604051601f8201601f191681016001600160401b03811182821017156158bd576158bd615a28565b604052919050565b60006001600160401b038211156158de576158de615a28565b50601f01601f191660200190565b600082198211156158ff576158ff615a12565b500190565b600060ff821660ff84168060ff0382111561592157615921615a12565b019392505050565b60008261594457634e487b7160e01b81526012600452602481fd5b500490565b600081600019048311821515161561596357615963615a12565b500290565b60008282101561597a5761597a615a12565b500390565b60005b8381101561599a578181015183820152602001615982565b83811115612b5f5750506000910152565b6000816159ba576159ba615a12565b506000190190565b600181811c908216806159d657607f821691505b60208210811415610f4f57634e487b7160e01b600052602260045260246000fd5b6000600019821415615a0b57615a0b615a12565b5060010190565b634e487b7160e01b600052601160045260246000fd5b634e487b7160e01b600052604160045260246000fd5b6001600160a01b0381168114613c6457600080fd5b8015158114613c6457600080fd5b6001600160e01b031981168114613c6457600080fdfeb309c40027c81d382c3b58d8de24207a34b27e1db369b1434e4a11311f154b5eddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3efb89cdd26cddd51301940bf2715f765b626b8a5a9e2681ac62dc83cc2db2530c0a264697066735822122056efe172c5db83c8227097a987fb132b5b7092d264a941d22a476c1cc3c09bff64736f6c63430008040033",
```

Comment:
these end up pretty heavy. I wonder if we should not "remove" them at build time...